### ISSUE TITLE
Default Redux dispatch() return value is being overridden/removed

### ISSUE THREAD

**Issue by jakereps** [samiskin/redux-electron-store#24]
The default return value for a Redux [`dispatch()`](http://redux.js.org/docs/api/Store.html#dispatch) is the action that was launched by the call. This package is breaking that default behavior, and `dispatch()` is only returning `undefined`.

To show a real-world application of this feature, here is a chunk of code from a project that is using the returned actions:

```JavaScript
fetchAPI(secretKey, method, url)
.then(json => json.plugins.map(plugin => dispatch(foundPlugin(plugin))))
.then(pluginActions => pluginActions.forEach(
    ({ plugin: { name, methodsURI, visualizersURI } }) => {
        dispatch(loadMethods(name, methodsURI));
        dispatch(loadVisualizers(name, visualizersURI));
    }));
```

Where the `json.plugins.map(dispatch())` was producing a list of the actions that were dispatched and would pass them along to the next step in the chain. After adding `redux-electron-store`, all that is returned is a list of `undefined`'s.

I tried hooking into the `postDispatchCallback()` to see if I could pump the action through using one of the available API methods, but it doesn't seem like that will work since telling that callback to return the action will just return it to the `doDispatch` method body, and not out of the `dispatch()` call itself.

**Comment by samiskin** [comment]
Ah good point, I did not know `dispatch` had a return value, so thanks for bringing it up.

If you could submit a PR to fix this that would be most helpful, since I'm pretty busy for the next week or so.

**Comment by jakereps** [comment]
I can definitely look into it. 

Thanks for this awesome package, by the way! It was a huge help with removing ipc "hacks" that threw objects between windows.

**Comment by samiskin** [comment]
Published in `v0.3.11` :+1:, thanks for contributing!

**Comment by jakereps** [comment]
Awesome! You're welcome! Thank you again, for this awesome package!
